libs/vue/src/builders/dev-server/builder.ts

```diff
@@ -41,13 +43,13 @@ export function runBuilder(
   options: DevServerBuilderSchema,
   context: BuilderContext
 ): Observable<BuilderOutput> {
+  const browserTarget = targetFromTargetString(options.browserTarget);
```

Comment:
I'd prefer this isn't moved. The project name is available at `context.target.project`.